scap-workbench-oscap.sh

```diff
@@ -74,7 +74,7 @@ while kill -0 $PID 2> /dev/null; do
     ret=$?
     if [ 0 -lt $ret -a $ret -lt 128 ]; then
         # If read failed & it was not due to timeout --> parents are gone.
-        kill -s SIGINT $PID 2> /dev/null
+        kill -TERM $PID 2> /dev/null
```

Comment:
I looked at it again and you are right, we should use TERM here.

However please change the `kill` syntax to `kill -s SIGTERM $PID`. According to the manpage that's more portable.